zerver/tests/test_channel_permissions.py

```diff
@@ -746,3 +748,241 @@ def test_change_stream_message_retention_days_requires_realm_owner(self) -> None
         self.assert_json_success(result)
         stream = get_stream("stream_name1", realm)
         self.assertEqual(stream.message_retention_days, 2)
+
+    def do_test_channel_administering_permission(
+        self, stream: Stream, property_name: str, new_value: str | int | bool
+    ) -> None:
+        realm = get_realm("zulip")
+        iago = self.example_user("iago")
+        shiva = self.example_user("shiva")
+        hamlet = self.example_user("hamlet")
+        prospero = self.example_user("prospero")
+        polonius = self.example_user("polonius")
```

Comment:
Can we give them names such as admin, owner, moderator, etc, depending on the use of that user in this test. It will make keeping track of why a user is used at a certain place without spending time thinking that iago is an admin.

Reply:
Updated this.